tests/cli/dorq/test_arg_resolvers.py

```diff
@@ -654,6 +655,464 @@ def return_wf(id, time_delay_in_sec: int):
             # Resolver should return the user's choice.
             assert resolved_run == selected_run
 
+    class TestResolveLogSwitches:
+        @staticmethod
+        @pytest.mark.parametrize(
+            "switches",
+            [
+                (False, False, False, True),
+                (False, False, True, False),
+                (False, False, True, True),
+                (False, True, False, False),
+                (False, True, False, True),
+                (False, True, True, False),
+                (False, True, True, True),
+                (True, False, False, False),
+                (True, False, False, True),
+                (True, False, True, False),
+                (True, False, True, True),
+                (True, True, False, False),
+                (True, True, False, True),
+                (True, True, True, False),
+                (True, True, True, True),
+            ],
+        )
```

Comment:
**non-blocking,suggestion**: seems like this fixture value sequence has an internal structure to it. Usually, it's more maintainable to define the fixture using that internal structure instead of pure copy-pasted values:
```suggestion
        @pytest.mark.parametrize(
            "switches",
            [
                (per_task, system, env_setup, other)
                for per_task in [True, False]
                for system in [True, False]
                for env_setup in [True, False]
                for other in [True, False]
                if any(per_task, system, env_setup, other)
            ],
        )
```